### DISCUSSION TITLE
[mime] Types for version 3 missing

### DISCUSSION THREAD

**Discussion by Fredi100** [DefinitelyTyped/DefinitelyTyped#60733] [Issues with a @types package]
There are no type definitions for version 3.0.0 of the `mime` npm package.
When using mime the IDE therefore incorrectly uses the version 2 definitions.

[a comment by an automated account is not shown]

**Comment by dhritzkiv** [comment]
Out of curiosity, does anything break? mime v3 should have the same API as v2

**Reply by Fredi100** [reply]
Sorry, my bad. It appears to fall back to the version 1 definition instead of using the v2.
This is the definition I get when using mime:
```Typescript
import { TypeMap } from "./index";

export default class Mime {
    constructor(mimes: TypeMap);

    lookup(path: string, fallback?: string): string;
    extension(mime: string): string | undefined;
    load(filepath: string): void;
    define(mimes: TypeMap): void;
}
```

Which results in following error when using `mime.getType(path)`:
```
TS2339: Property 'getType' does not exist on type 'Mime'.
```
